sdk/appconfiguration/Azure.ApplicationModel.Configuration/tests/ConfigurationWatcherTests.cs

```diff
@@ -17,7 +17,7 @@ public class ConfigurationWatcherTests
         [Test]
         public async Task Helpers()
         {
-            ConfigurationClient client = TestEnvironment.GetClient();
+            ConfigurationClient client = new ConfigurationClient(TestEnvironment.GetClientConnectionString());TestEnvironment.GetClientConnectionString();
```

Comment:
> new ConfigurationClient(TestEnvironment.GetClientConnectionString());TestEnvironment.GetClientConnectionString(); [](start = 40, length = 114)

seems like an accidental duplication of TestEnvironment.GetClientConnectionString